crates/cxx-qt-lib/build.rs

```diff
@@ -271,5 +276,6 @@ fn main() {
     // GCC + Clang
     builder.flag_if_supported("-std=c++17");
 
+    println!("cargo:rerun-if-env-changed=compile");
```

Comment:
this looks like a trace line ?

Reply:
Isn't it useful to have markers at least in the higher level class to follow the steps ?

Reply:
```suggestion
```